custom-modifiers/waitforimagemodifier/src/main/java/WaitForImageModifier.java

```diff
@@ -0,0 +1,85 @@
+import com.cognifide.aet.communication.api.metadata.CollectorStepResult;
+import com.cognifide.aet.job.api.collector.CollectorJob;
+import com.cognifide.aet.job.api.exceptions.ParametersException;
+import com.cognifide.aet.job.api.exceptions.ProcessingException;
+import org.openqa.selenium.*;
+import org.openqa.selenium.support.ui.ExpectedCondition;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.FluentWait;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Map;
+import java.util.NoSuchElementException;
+import java.util.concurrent.TimeUnit;
+
+public class WaitForImageModifier implements CollectorJob {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(WaitForImageModifier.class);
+
+    static final String NAME = "wait-for-image";
```

Comment:
What do you think about renaming it to `wait-for-image-finished-loading` or `wait-for-image-completion` or `wait-for-image-is-complete`?
Please adjust the package and class name if you change the name.